### PR TITLE
BOAC-1317: gives every user their own blur mode

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.ets.berkeley.edu/jira/browse/BOAC-1317

In a follow-up PR, I will move the toggle from the admin page to the top right menu under the user's name.

### CODE REVIEW

File: config/default.py [hunk 1]
```diff
@@ -31,12 +31,6 @@
 # Base directory for the application (one level up from this config file).
 BASE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
 
-# When BOAC is in demo-mode all student names and SIDs are blurred in the UI
-DEMO_MODE = {
-    'blur': False,
-    'betaTesting': False,
-}
```
Comment (johncrossman) [inline config/default.py]: I'm glad to see this go. Don't forget to update BOAC release instructions doc.

File: scripts/db/schema.sql [hunk 2]
```diff
@@ -88,7 +88,8 @@ CREATE TABLE authorized_users (
     updated_at timestamp with time zone NOT NULL,
     id integer NOT NULL,
     uid character varying(255) NOT NULL,
-    is_admin boolean
+    is_admin boolean,
+    blur_mode boolean DEFAULT false NOT NULL
```
Comment (johncrossman) [inline scripts/db/schema.sql]: @lyttam As mentioned, I prefer column name `in_demo_mode`

**Comment by johncrossman** [comment]
Thank you @lyttam. Good work. I made a couple suggestions.

File: config/default.py (hunk 1 above)
Comment (lyttam) [reply config/default.py]: Done, thanks for the reminder.

File: boac/models/authorized_user.py [hunk 3]
```diff
@@ -36,6 +36,7 @@ class AuthorizedUser(Base, UserMixin):
     id = db.Column(db.Integer, nullable=False, primary_key=True)  # noqa: A003
     uid = db.Column(db.String(255), nullable=False, unique=True)
     is_admin = db.Column(db.Boolean)
+    blur_mode = db.Column(db.Boolean, nullable=False)
```
[the remark this answers is not in the document]
Comment (lyttam) [reply boac/models/authorized_user.py]: @johncrossman I took your suggestion and amended.

**Comment by johncrossman** [comment]
+1  Thx @lyttam

**Pull request merged**